src/averageAges.js

```diff
@@ -13,13 +13,28 @@
  * @param {number} century - optional
  *
  * @return {number}
+ *
  */
+const calculateAverageAge = function(array) {
+  const length = array.length;
+
+  return array.reduce((sum, age) => sum + age, 0) / length;
+};
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const ages = people
+    .filter((person) => century
+      ? person.sex === 'm'
+      && century === Math.ceil(person.died / 100)
+      : person.sex === 'm')
```

Comment:
move common condition out of ternary 
```suggestion
    .filter((({ sex, died })) => sex === 'm' && (!century || Math.ceil(died / 100)))
```